Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphListener/MicrosoftGraphListener.yml

```diff
@@ -818,6 +818,37 @@ script:
     - contextPath: MSGraphMail.Folders.UnreadItemCount
       description: The number of unread emails in the folder.
       type: number
+  - arguments:
+    - description: The ID of the parent folder.
+      name: parent_folder_id
+      required: true
+    - defaultValue: '20'
+      description: The maximum number of mail folder lists to return. Default is 20.
+      name: limit
+    - description: flag for rate limit retry.
+      name: ran_once_flag
+    description: Returns the folder list under the specified folder.
+    name: msgraph-mail-list-child-folders
+    outputs:
+    - contextPath: MSGraphMail.Folders.ChildFolderCount
+      description: The number of child folders.
+      type: Number
+    - contextPath: MSGraphMail.Folders.DisplayName
+      description: The folder display name.
+      type: String
+    - contextPath: MSGraphMail.Folders.ID
+      description: The folder ID.
+      type: String
+    - contextPath: MSGraphMail.Folders.ParentFolderID
+      description: The parent folder ID.
+      type: String
+    - contextPath: MSGraphMail.Folders.TotalItemCount
+      description: The total number of email messages in the folder.
+      type: Number
+    - contextPath: MSGraphMail.Folders.UnreadItemCount
+      description: The number of unread email messages in the folder.
+      type: Number
+    polling: true
```

Comment:
```suggestion
```
Is it on purpose?
Because the function [here](https://github.com/demisto/content/blob/master/Packs/ApiModules/Scripts/MicrosoftGraphMailApiModule/MicrosoftGraphMailApiModule.py#L1778) doesn't looks like a polling function.

Reply:
The code is largely from the other integration in this pack and it had polling for this command: https://github.com/demisto/content/blob/d03739c8c6b73823d390494b357e9c4785102e12/Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.yml#L555

I'll have to dig into the shared API code to see what's going on with it and validate.